package.json

```diff
@@ -79,6 +79,7 @@
     "react-virtualized": "^9.20.1",
     "reflux": "0.4.1",
     "scroll-to-element": "^2.0.0",
+    "sentry-dreamy-components": "getsentry/dreamy-components",
```

Comment:
Unrelated to this PR, but wanted to leave this here.

@Chrissy is this package "stable" enough to publish to npm? imo we should rename to use `@sentry` namespace and publish so we can take advantage of versioning contraints.

Reply:
@billy If this goes out and it's successful then yah I would be willing to publish to NPM and rename all these imports. When I used it on the static site I had to make a ton of commits to the original package, and thought I might have to do so here too.